site/docs/v1/tech/email-templates/templates-replacement-variables.adoc

```diff
@@ -0,0 +1,610 @@
+[/#if]
+
+
+- FusionAuth Admin
+----
+
+=== Replacement Variables
+
+[.api]
+[field]#application# [type]#[Application]# [since]#Available since 1.21.0#::
+The Application object, see the Application API for field definitions.
+
+[field]#registration# [type]#UserRegistration#]::
+The User Registration object, see the Registration API for field definitions of a User.
+
+[field]#tenant# [type]#[Tenant]# [since]#Available since 1.18.2#::
+The Tenant object, see the Tenant API for field definitions of a Tenant.
+
+[field]#user# [type]#[User]#::
+The User object, see the User API for field definitions of a User.
+
+[field]#verificationId# [type]#[String]#::
+The verification Id intended to be used by the link:/docs/v1/tech/apis/registrations#verify-a-user-registration[Verify a User Registration] API.
+
+[field]#verificationOneTimeCode# [type]#[String]#::
+The verification One Time Code to be used with the Gated Registration workflow. The user enters this code to verify their email.
+
+== Setup Password
+
+.HTML
+[source,text]
+----
+Your account has been created and you must setup a password. Click on the following link to setup your password.
+<p>
+  <a href="http://localhost:9011/password/change/${changePasswordId}?tenantId=${user.tenantId}">
+    http://localhost:9011/password/change/${changePasswordId}?tenantId=${user.tenantId}
+  </a>
+</p>
+- FusionAuth Admin
+----
+
+.Text
+[source,text]
+----
+Your account has been created and you must setup a password. Click on the following link to setup your password.
+
+http://localhost:9011/password/change/${changePasswordId}?tenantId=${user.tenantId}
+
+- FusionAuth Admin
+----
+
+=== Replacement Variables
+
+[.api]
+[field]#application# [type]#[Application]# [since]#Available since 1.21.0#::
+The Application object, see the Application API for field definitions.
++
+*Note*:
+This object may not be available depending upon when this template is constructed. If you utilize this object in your template, ensure you first check to see if it is defined.
+
+[field]#changePasswordId# [type]#[String]#::
+The change password Id intended to be used by the link:/docs/v1/tech/apis/users#change-a-users-password[Change a User's Password] API.
+
+[field]#tenant# [type]#[Tenant]# [since]#Available since 1.18.2#::
+The Tenant object, see the Tenant API for field definitions of a Tenant.
+
+[field]#user# [type]#[User]#::
+The User object, see the User API for field definitions of a User.
+
+== Two Factor Authentication
+
+.HTML
+[source,text]
+----
+<p>
+  To complete your login request, enter this one-time code code on the login form when prompted.
+</p>
+<p>
+  <strong>${code}</strong>
+</p>
+
+- FusionAuth Admin
+----
+
+.Text
+[source,text]
+----
+To complete your login request, enter this one-time code code on the login form when prompted.
+
+${code}
+
+- FusionAuth Admin
+----
+
+=== Replacement Variables
+
+[.api]
+[field]#application# [type]#[Application]# [since]#Available since 1.28.0#::
```

Comment:
I didn't think we've made it to 1.28 yet.  Isn't this 1.26

Reply:
This was added based on dan's note that this will be in 1.28 #647 (comment).

Any thoughts on the above ☝🏻 https://github.com/FusionAuth/fusionauth-site/pull/673/#issuecomment-843582016
specifically `consents replacement object`?